spec/javascripts/Settings/components/Common/TextInputGroup.spec.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+import { mount } from 'enzyme'
```

Comment:
This shouldn't really be in this PR.

Reply:
https://github.com/3scale/porta/pull/1655#discussion_r376983311

Reply:
Right